web-console/developer/add/add.md

```diff
@@ -1,29 +1,32 @@
 # Add
-Add is a primary nav item in the OpenShift Developer perspective.
+Add is a primary navigation item in the OpenShift Developer perspective.
 
-![Add primary nav](img/dev-add.png)
+![Add primary nav](img/Add-AltA.png)
 
 The Add page is a card view with a number of cards enables the user to add to their project in different ways.  
```

Comment:
@bgliwa01  can we change this sentence to something more like (feel free to reword though!): 

"The Add page enables the user to create an application, component or service.   Upon clicking a card, the user is brought to the appropriate flow to add the item to their project.

The basic visuals and behavior of this card view is consistent with that of the developer catalog and operator hub."